eos/db/migrations/upgrade16.py

```diff
@@ -0,0 +1,14 @@
+"""
+Migration 16
+
+- Alters fits table to introduce notes attribute
+"""
+
+import sqlalchemy
+
+def upgrade(saveddata_engine):
+    # Update fits schema to include notes attribute
+    try:
+        saveddata_engine.execute("SELECT notes FROM fits LIMIT 1")
+    except sqlalchemy.exc.DatabaseError:
+        saveddata_engine.execute("ALTER TABLE fits ADD COLUMN notes STRING;")
```

Comment:
`STRING` isn't exactly a data type in SQLite.

Historically we've been using `VARCHAR` (which just causes it to use `TEXT`).  You probably want to use `TEXT`, though `BLOB` might be an option as well.

https://www.sqlite.org/datatype3.html

I would say use either `VARCHAR` or `TEXT`.